### PR TITLE
MINOR: Adds projection setter to BoundsGenerator

### PR DESCRIPTION
**Pull request by FraukeF**
Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: @here/harp-mapview/test/BoundsGeneratorTest.ts [hunk 1]
```diff
@@ -106,6 +106,18 @@ describe("BoundsGenerator", function() {
         assert.isEmpty(geoPolygon);
     });
 
+    it("generates empty polygon for spherical projection,  after projection changes", function() {
+        mapView?.lookAt(lookAtParams);
+        mapView?.renderSync(); // render once to update camera parameter
```
Comment (ninok) [inline @here/harp-mapview/test/BoundsGeneratorTest.ts:111]: this should not be necessary. LookAt is already updating everything. If not it is a bug and should be fixed

Comment (FraukeF) [reply @here/harp-mapview/test/BoundsGeneratorTest.ts:111]: It is actually not updating, but this is also not a bug in MapView, but an issue of how the stubbed MapView in the test is created. 
As we are running also in node context, basically the MapView is not complete and the Renderer and AnimationFrameRequest etc are just stubbed.
The MapView in this Test is not really needed but just used as a helper to set the camera parameters, that are used in the BoundsGenerator.

Comment (ninok) [reply @here/harp-mapview/test/BoundsGeneratorTest.ts:111]: I think we should fix the near/far plane computation to be updated when look at is called and not only in the next animation frame. At least we could compute the near and far plane based on the camera and min/max elevation settings.
When the actual rendering happens we can then refine it based on the actual min/max elevation of the visible tiles.

Comment (FraukeF) [reply @here/harp-mapview/test/BoundsGeneratorTest.ts:111]: I ll create a ticket for it

**Pull request merged**